Frameworks/Foundation/NSDictionary.mm

```diff
@@ -99,8 +99,11 @@ - (NSArray*)allObjects {
         return nil;
     }
 
-    StrongId<NSArray> keyArray = [_keyEnumerator allObjects];
-    StrongId<NSMutableArray> toRet = [NSMutableArray new];
+    StrongId<NSArray> keyArray;
+    keyArray.attach([_keyEnumerator allObjects]);
+
+    StrongId<NSMutableArray> toRet;
```

Comment:
>toRet [](start = 29, length = 5)

is this really need to be wrapped in StrongId? given it's already auto released (as per [NSMutableArray new]) and it's being returned.